### DISCUSSION TITLE
Getting a packet capture of the bicking process

### DISCUSSION THREAD

**Discussion by GerryDazoo** [GerryDazoo/Slinger#340] [General]
I don't have a ProHD :-( Now that we can unbrick the ProHD,  we really need a packet capture of the bricking process. Can anybody out there open up their firewall rules and get a packet capture of all the traffic to/from the slingbox? Yes, I know it's a pain to unbrick but...

PS. I really do need a ProHD for development/testing purposes. Anybody want to send me one? I'm in Canada.

-Gerry

**Comment by Sure4Thing** [comment]
Sure hit me up with where you want it delivered!  Do you need a power supply too or just the box?

**Reply by GerryDazoo** [reply]
Many thanks. 

Yes, I need the power supply.

**Reply by Sure4Thing** [reply]
Feel free to remove your address now

**Comment by Sure4Thing** [comment]
Shipped VIA UPS

**Reply by GerryDazoo** [reply]
Many thanks. May the Slingbox gods smile upon you and everybody else using Slinger.

**Comment by GerryDazoo** [comment]
Sure4Thing,
I went to UPS to track the package and it doesn't look like it's coming my way. ?!

**Comment by Sure4Thing** [comment]
Well, it's international shipping so I'm sure it has to go to a hub first
and then it'll probably head out into Canada.

**Reply by GerryDazoo** [reply]
Mr. UPS says the package was delivered to someone in Kentucky yesterday. :-(
Maybe I have the wrong tracking number? 1Z14V5350332546558

**Comment by socelillyhammer** [comment]
@GerryDazoo  Im not a tech guy but I left my slinbox 500s on and researched this post a bit too late. If you can help in any way I'll be more than happy to pay $500 per each box to unbrick and help me out getting the boxes working again.

**Reply by GerryDazoo** [reply]
As far a we know the 350/500/M1 have not been bricked. What is your issue?
PS. Just to be safe you should block out going Internet access for your boxes

**Reply by socelillyhammer** [reply]
I have no clue how to do that with a verizon fios router.  I did not login to newwatch.slingbox in time to grab my passwords. So I dont know if there is a work around. But basically I'm sitting with 2 boxes now that are just taking up space.

**Reply by robosmith** [reply]
According to the guy in this video: https://www.youtube.com/watch?v=EUyQfpzHRkg
a factory reset sets your admin pw to "admin" and you just set it up as normal.
I'm hoping that works, too, because I got a 500 off ebay, but haven't tried it with Slinger yet.

**Comment by Sure4Thing** [comment]
It got delivered to customs in Kentucky my guess is the next step is that
it'll go to you next.

I'll see what I can find out

**Reply by GerryDazoo** [reply]
Could be, I just noticed the delivery time 11PM, So I'm pretty sure that wasn't somebody's house. I'll hang tight.

**Comment by Sure4Thing** [comment]
https://m.17track.net/en/track-details?nums=UPAAH000000039127645

**Reply by GerryDazoo** [reply]
That's a lot more detail than UPS is showing me. I'll use that link from now on.

**Comment by Sure4Thing** [comment]
17track is great.  It ties into all of the shipping services at once.
Really useful for tracking stuff from Asia and such.

**Comment by GerryDazoo** [comment]
Yeah! I got it. Many thanks.

**Comment by Sure4Thing** [comment]
HUZZAH I SAY HUZZAH
